docs/core/tools/sdk-errors/netsdk1182.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "NETSDK1182: Targeting .NET 6.0 or higher in Visual Studio 2019 is not supported."
+description: Learn about the 'Targeting .NET 6.0 or higher in Visual Studio 2019 is not supported' error message.
+author: tdykstra
+ms.author: tdykstra
+ms.topic: error-reference
+ms.date: 07/08/2022
+f1_keywords:
+- NETSDK1182
+---
+# NETSDK1182: Targeting .NET 6.0 or higher in Visual Studio 2019 is not supported
+
+NETSDK1182 indicates that you're trying to open a .NET 6+ project in Visual Studio 2019. The full error message is similar to the following example:
```

Comment:
I believe you would also get this error message if you build from the command line using `msbuild` from VS 2019.